packages/server/src/inMemoryInstances.ts

```diff
@@ -1,24 +1,27 @@
 import { Socket } from "socket.io";
 import Instance from "./models/Instance";
 
-export type UserSlot = { slot_index: number; client: Socket | null | undefined };
-export type InstanceUser = { id: string; client_index: number; name: string };
+
+export type InstanceSlot = { slot_index: number; client: Socket | null | undefined };
+export type ConnectedClient = { id: string; client_index: number; name: string };
 
```

Comment:
This change is to clarify the purpose of these types and they're use throughout the codebase.

Reply:
@laubsauger -> This is leftover from `socketdrawing-server` project.

After piecing this apart, the only reason I see these as being separate is that we want to keep track of connected clients and those clients that have been given a slot.

That said, I think that we disconnect a client if there is not slot for them. See https://github.com/laubsauger/osc.link/blob/d610ff1efd4d488ab54a7a6138902722cb706827/packages/server/src/utils.ts#L16-L22